tests/specs/mod.rs

```
@@ -256,19 +256,10 @@ fn run_test(test: &CollectedTest<serde_json::Value>) -> TestResult {
     if metadata.ignore {
       TestResult::Ignored
     } else if let Some(repeat) = metadata.repeat {
-      TestResult::SubTests(
-        (0..repeat)
-          .map(|i| {
-            let diagnostic_logger = diagnostic_logger.clone();
-            SubTestResult {
-              name: format!("run {}", i + 1),
-              result: TestResult::from_maybe_panic(AssertUnwindSafe(|| {
-                run_test_inner(&metadata, &cwd, diagnostic_logger);
-              })),
-            }
-          })
-          .collect(),
-      )
+      for _ in 0..repeat {
+        run_test_inner(&metadata, &cwd, diagnostic_logger.clone());
```

Better to fail fast when repeating a test.